app/services/import_post_service.rb

```diff
@@ -17,9 +17,38 @@ def call
       post.permalink_url = content['permalink_url']
       post.user = ImportUserService.call(content['from'])
     end
+    import_comments(post, content['comments']['data']) if content['comments'].present?
+    import_reactions(post, content['reactions']['data']) if content['reactions'].present?
+
+    return post
   end
 
   private
 
   attr_reader :content
+
+  def import_comments(post, comments)
```

Comment:
lets separate this into `import_comment_service` 
and the same for `import_reactions` 

so that we have a separate service that import each entity type.

Reply:
Thank you , let me try to do it by myself first.
and, Do you think we need to add counter in Post for every type of reaction or just leave it as it is now ?

Reply:
I think we can deliver this first and think about separate counter for every reaction type later, as this could be done separately, we don't have to make both at once